### ISSUE TITLE
version 1.4.2 does not escape HTML correctly

### ISSUE THREAD

**Issue by sebbASF** [rubys/wunderbar#16]
```ruby
#!/usr/bin/env ruby

require 'nokogiri'
require 'wunderbar'
require 'wunderbar/version'

if __FILE__ == $0
  h3='email "Some One" <someone@gmail.com>'
  vers = Wunderbar::VERSION::STRING
  _html do
    _h2 vers
    _h3 h3
    _ h3
  end
end
```

The above test script generates:
```

<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8"/>
  </head>

  <body>
    <h2>1.4.2</h2>
    <h3>
      email "Some One" 
      <someone></someone>
    </h3>
    email "Some One" 
    <someone></someone>
  </body>
</html>
```

Whereas it should generate:

```
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8"/>
  </head>

  <body>
    <h2>1.3.4</h2>
    <h3>email "Some One" &lt;someone@gmail.com&gt;</h3>
    email "Some One" &lt;someone@gmail.com&gt;
  </body>
</html>
```

It looks like some escaping has been lost between 1.3.4 and 1.4.2

**Comment by sebbASF** [comment]
I think the issue may have been caused by the removal of various html_safe? checks.

AFAICT, that is not the same as the ruby issue with $SAFE/untaint

**Comment by sebbASF** [comment]
Should be fixed by PR#20

**Comment by sebbASF** [comment]
Fixed in current code
